lib/msfenv.rb

```diff
@@ -17,3 +17,5 @@
   require config.join('environment')
 end
 require 'msf_autoload'
+
+MsfAutoLoad.instance
```

Comment:
Maybe this could turn into
```suggestion
MsfAutoLoad.setup_environment!
```
at some point